db/init/00-database.sql

```diff
@@ -234,22 +250,51 @@ CREATE TRIGGER session_delete_session_result BEFORE DELETE
 ON base.session FOR EACH ROW EXECUTE PROCEDURE
 base.delete_children('session_result', 'session_id');
 
+ALTER TABLE base.session ENABLE ROW LEVEL SECURITY;
+
 
 
 /*Create table session result*/
 CREATE TABLE base.session_result (
-    id SERIAL PRIMARY KEY,
-    alert_operator TEXT NOT NULL,
-    alert_threshold FLOAT NOT NULL,
-    nb_records INTEGER NOT NULL,
-    nb_records_alert INTEGER NOT NULL,
-    nb_records_no_alert INTEGER NOT NULL,
-    created_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
-    session_id INTEGER NOT NULL REFERENCES base.session(id)
+    id SERIAL PRIMARY KEY
+  , alert_operator TEXT NOT NULL
+  , alert_threshold FLOAT NOT NULL
+  , nb_records INTEGER NOT NULL
+  , nb_records_alert INTEGER NOT NULL
+  , nb_records_no_alert INTEGER NOT NULL
+  , user_group TEXT NOT NULL
+  , created_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+  , session_id INTEGER NOT NULL REFERENCES base.session(id)
 );
 COMMENT ON TABLE base.session_result IS
 'Session results contain a summary of indicators execution.';
 
+ALTER TABLE base.session_result ENABLE ROW LEVEL SECURITY;
+
+
+/*Create table user*/
+CREATE TABLE base.user (
+    id SERIAL PRIMARY KEY
+  , email TEXT NOT NULL UNIQUE
+  , flag_active BOOLEAN DEFAULT TRUE
+  , oauth_type TEXT NOT NULL
```

Comment:
Do we keep this as text or do we plan to maybe add a table for all OAuth providers? The table could then also contain URLs for OAuth, etc?
What do you think @alexisrolland ?

Reply:
later on this should be done but for now this should be sufficient